src/secp256k1/src/num.h

```diff
@@ -7,12 +7,12 @@
 #ifndef SECP256K1_NUM_H
 #define SECP256K1_NUM_H
 
-#ifndef USE_NUM_NONE
-
 #if defined HAVE_CONFIG_H
 #include "libsecp256k1-config.h"
 #endif
 
+#ifndef USE_NUM_NONE
+
```

Comment:
why is this necessary?

Reply:
because libsecp256k1-config.h might define USE_NUM_NONE

Reply:
I've configured the android version of libsecp256k1 to have no architecture specific optimizations because the android code is not performance sensitive and bringing architecture-specific opts will not be easy since android runs on many architectures.  This no-opts configuration is likely extremely rarely used in libsecp256k1.